var/spack/repos/builtin/packages/xz/package.py

```diff
@@ -112,17 +112,31 @@ def msbuild_args(self):
     def install(self, pkg, spec, prefix):
         with working_dir(self.build_directory):
             # Ensure we have libs directory
-            mkdirp(prefix.lib)
+            if not os.path.isdir(prefix.lib):
+                mkdirp(prefix.lib)
+            if not os.path.isdir(prefix.bin):
+                mkdirp(prefix.bin)
```

Comment:
`mkdirp` works whether or not dir has already been made: please remove from `if` statement.